Makefile

```diff
@@ -11,25 +11,25 @@ ALL: $(CMDS)
 
 ui: ui/bindata.go
 
-node_modules:
+node_modules/build:
```

Comment:
I'm not well-versed in Make, but is it possible to drop a comment here explaining why `node_modules/build` is the target? That feels like one of those things that would get lost to time otherwise.